docs/debugging.qmd

```diff
@@ -0,0 +1,224 @@
+---
+title: "Debugging"
+order: 1
+---
+
+# Debugging data issues
+
+### DON'T PANIC!
+
+Many issues can arise in the Metro galaxy, from the shallowest part of the frontend to the deepest depths of the backend. However, these issues generally fall into [two broad categories](#categories-of-failure):
+
+- [Data is missing](#data-is-missing)
+- [Data is incorrect](#data-is-incorrect)
+- Appendices
+    - [More on Airflow](#more-on-airflow)
+    - [View an entity in the Councilmatic database](#view-an-entity-in-the-councilmatic-database)
+    - [Inspect the scraper logs](#inspect-the-scraper-logs)
+
+## Categories of failure
+
+This section expounds on common culprits in our three categories of failure: missing data, incorrect data, and issues with metadata. The culprits are ordered from most to least likely, therefore we suggest moving through the category relevant to your problem in order until you identify the issue.
+
+### Data is missing
+
+#### Scrape failures
+
+Our scrapers are brittle by design, i.e., they will generally fail if they try to ingest data that is formatted incorrectly. If there has been a scrape failure, there should be a corresponding exception in [the `scrapers-lametro` project in Sentry](https://datamade.sentry.io/projects/scrapers-lametro/?project=4504447849201664).
+
+Common exceptions include `OperationalError`, which indicates that something went awry with the server (e.g., not enough disk space) and `ScraperError`, which indicates a particular issue with the one of the scrapers.
+
+If you don't see an issue in Sentry, but you have reason to believe there was an error (or that our integration with Sentry is faulty), you can [follow our documentation for finding errors in the scraper logs](#inspect-the-scraper-logs).
+
+If there is not an error in Sentry or in the scraper logs, then the scrape ran. 
+
+#### Inaccurate timestamps in the Legistar API
+
+If scrapes are running, by far the most common source of issues is the scraper failing to capture changes to a bill or event. The root issue is that we rely on timestamps that should indicate an update to determine which bills and events to scrape. In reality, these timestamps do not always update when a change is made to an event or bill in Legistar. We have a couple of strategies to get around this:
+
+1. During [the Friday support window](https://github.com/datamade/la-metro-dashboard/blob/ac416e5e03f6a97fb9b0c6112093c679cefb0d1c/dags/constants.py#L41-L59), we scrape all events and bills at the top of every hour.
```

Comment:
Should we remove references to the Friday support window?

Reply:
Yes! Done.